index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>My Portfolio</title>
+</head>
+
+<body>
+  <div class="container">
+    <header class="navbar">
+      <a href="#">Dan Logo</a>
+      <span> <img src="images/burger.png" alt="toggle icon" /> </span>
+    </header>
+    <section class="header">
+      <div class="header-title">
+        <h1>Hey there. I'm Dan Elifeleti</h1>
+        <h2 class="green">I'm a software developer</h2>
+      </div>
+      <div class="header-content">
+        <p>
+          I can help you build a product , feature or website Look through
+          some of my work and experience! If you like what you see and have a
+          project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="icon-style">
+        <ul class="icons">
+          <li>
+            <a href="#"><img src="images/git-logo.png" alt="cat icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/linkedin.png" alt="linkldn icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/vector.png" alt="two icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/twitter-logo.png" alt="twitter icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/m-logo.png" alt="m icon" /></a>
+          </li>
+        </ul>
+      </div>
+
+      <div class="anchor-arrow">
+        <img src="images/anchor-arrow.png" alt="arrow" />
+      </div>
+    </section>
+    <section class="project-container">
+      <div class="project-title">
+        <h2>My Recent </h2>
+      </div>
+      <div class="project-list">
+        <ul>
+          <li class="list-item">
+            <div class="image-project"></div>
+            <div class="image-style">
+              <h2>Tonic</h2>
+              <div class="buttons">
+                <div>PHP</div>
+                <div>css</div>
+                <div>JavScript</div>
+                <div>html</div>
+              </div>
```

Comment:
- [ ] Your design is looking great! However, this list should use `ul` and `li` tags instead of `div`, please do this for every card.